node/engine/store/durablestore.go

```diff
@@ -589,18 +589,19 @@ func (ds *DurableStore) SetVoucherInfo(channelId types.Destination, v payments.V
 	})
 }
 
-func (ds *DurableStore) GetVoucherInfo(channelId types.Destination) (v *payments.VoucherInfo, ok bool) {
+func (ds *DurableStore) GetVoucherInfo(channelId types.Destination) (*payments.VoucherInfo, error) {
+	v := &payments.VoucherInfo{}
 	err := ds.vouchers.View(func(tx *buntdb.Tx) error {
 		vJSON, err := tx.Get(channelId.String())
 		if err != nil {
-			return nil
+			return err
```

Comment:
Changing this line allows us to better handle the non-existent channel [here](https://github.com/statechannels/go-nitro/blob/5ec7d46213d97cf631badd773d82bab3fcf9dfe6/payments/voucher_manager.go#L55).